src/tools/illink/test/Mono.Linker.Tests.Cases/Reflection/MemberUsedViaReflection.cs

```diff
@@ -144,9 +144,9 @@ private class SimpleType
 			[Kept]
 			public int memberKept {
 				[Kept]
-				get { return field; }
+				get { return @field; }
```

Comment:
I'd rename this to "fieldKept" to match the rest of the test - and it will resolve the problem as well.